ui/src/routes/(platform)/dashboard/+layout@(platform).svelte

```diff
@@ -270,28 +215,42 @@
   }
 
   :global(.bx--side-nav__divider) {
-    background-color: var(--cds-ui-01);
+    background-color: var(--cds-ui-01) !important;
   }
 
+  /* Link item hover color */
   :global(.bx--side-nav__link):hover {
     background-color: var(--cds-ui-01) !important;
   }
 
   :global(.bx--side-nav__submenu) {
-    cursor: default;
+    cursor: default !important;
   }
 
   /* Remove hover highlight on hover for the Sidenav header */
   :global(.bx--side-nav__submenu:hover) {
-    background-color: transparent;
+    background-color: transparent !important;
   }
 
   /* Sidenav active/ selected link background color */
-  :global(a.bx--side-nav__link--current) {
+  :global(a.bx--side-nav__link[aria-current='page']) {
     background-color: var(--cds-ui-01) !important;
   }
 
-  :global(a.bx--side-nav__link--current > .bx--side-nav__link-text) {
+  :global(a.bx--side-nav__link[aria-current='page']:focus) {
```

Comment:
Briefly chatted about this with @TristanHoladay earlier, do we need these styles to be `global`?